drivers/mdio/mdio_i2c.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright The Zephyr Project Contributors
```

Comment:
is this meaningful? 

cc @kartben

Reply:
https://docs.zephyrproject.org/latest/contribute/guidelines.html#copyrights-notices